shared/controlflow/codeql/controlflow/Guards.qll

```diff
@@ -234,16 +234,16 @@ signature module InputSig<LocationSig Location> {
    */
   predicate parameterMatch(ParameterPosition ppos, ArgumentPosition apos);
 
-  /** A non-overridable method with a boolean return value. */
-  class BooleanMethod {
+  /** A non-overridable method. */
+  class NonOverridableMethod {
```

Comment:
I'm not sure we should use the `overridable` terminology here; how about simply `Callable` (and `Call` below)?